### DISCUSSION TITLE
Lesson 4- Error reading in directory

### DISCUSSION THREAD

**Discussion by EdenSwack** [smartcontractkit/full-blockchain-solidity-course-py#1735] [Q&A] [answered]
Within the deploy.py file and running the following code: 

`with open("./SimpleStorage.sol", "r") as file:
    simple_storage_file = file.read()
    print(simple_storage_file)
`
I get the following error:
`FileNotFoundError: [Errno 2] No such file or directory: './SimpleStorage.sol'`

**UPDATE** 
I copied the relative path of SimpleStorage.sol and pasted it instead but because i'm already in this directory shouldn't also work as above?

<img width="668" alt="Screen Shot 2022-10-05 at 5 09 37" src="https://user-images.githubusercontent.com/93931254/193964191-876b50fd-c253-4673-9e12-acd880bc8d40.png">

**Comment by cromewar** [comment, marked as answer]
Hello @EdenSwack 
First of all, be sure you have saved the changes on `Simple Storage` contract.
Secondly try to find any indentation error on the code and finally restart vscode, this might help.

Additional tip, be sure to be on the correct folder on the terminal when running the code.
